### ISSUE TITLE
MoBi 12: Hide some standard warnings?

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/MoBi#1112]
When I create a simulation based on a **small molecules spatial structure** (which does not include **Endosome** compartments) - I get a warning for each organ and each protein, that the initial condition cannot be applied to the `<Organ>|Endosome` container.
The warning is correct; however it will be the case whenever I use expression profiles exported from PK-Sim in combination with small molecules spatial structure.
E.g. below for 5 Proteins I get 135 Warnings (27 Organs * 5).

Would be nice to hide those warnings per default.
![grafik](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25061876/d8530eb0-43f5-4eda-9f5f-ab7736edd3b7)

**Comment by PavelBal** [comment]
@rwmcintosh We also discussed this, let's see how this impacts performance and decide what to do.

**Comment by rwmcintosh** [comment]
I'm having an issue with profiling right now and am waiting on JetBrains to get back to me, so I don't have an accurate measurement of the performance hit yet. It seems negligible. We have very recently made an improvement in how the VisitorInvoker works so we do not have the performance hit we once did when showing a large number of notifications.

I'd say there's probably a better reason to hide them though. If you have warnings that are not really warnings, then other 'real' warnings are going to get ignored. Just ask anyone who has worked with C/C++ compilers.

**Comment by msevestre** [comment]
Hide them not for performance but because it's noise that we create by default out of PKSim

**Comment by rwmcintosh** [comment]
Should this be specific to paths that end with 'Endosome' in MoBi? Or more general 'missing initial conditions container'?

**Comment by Yuri05** [comment]
> Should this be specific to paths that end with 'Endosome' in MoBi? Or more general 'missing initial conditions container'?

I would start with Endosome only.

**Comment by Yuri05** [comment]
Created a new issue for the 3 remaining warnings: https://github.com/Open-Systems-Pharmacology/MoBi/issues/1758
